simpler_eda/numerical_eda.py

```diff
@@ -3,52 +3,51 @@
 import pandas as pd
 
 
-
 def numerical_plot(
     data,
-    xval = None,
-    yval = None,
-    plot_type = "scatter",
-    color = None,
-    title = None ,
-    font_size = 10,
-    colour_scheme = "yellowgreenblue",
```

Comment:
Having a second thought, this is coloring for the different categories in a scatter plot. A sequential color frame may be hard to differentiate. tableau20 might be a good alternative.

Reply:
Changed to categorical scheme